core/pkg/observability/sentry.go

```diff
@@ -12,21 +12,25 @@ import (
 	"github.com/getsentry/sentry-go"
 )
 
-const sentryDsn = "https://0d0c6674e003452db392f158c42117fb@o151352.ingest.sentry.io/4505513612214272"
+const defaultSentryDsn = "https://0d0c6674e003452db392f158c42117fb@o151352.ingest.sentry.io/4505513612214272"
```

Comment:
naming: this should be `defaultSentryDSN` following Go conventions (DSN = Data Source Name)